tests/Controller/EnMarche/CommitteeControllerTest.php

```diff
@@ -285,6 +287,81 @@ public function testAuthenticatedCommitteeHostCanShowCommitteePage()
         $this->assertTrue($this->seeMessageForm($crawler));
     }
 
+    public function testNoEditLinkWithAnonymousUser()
+    {
+        $crawler = $this->client->request(Request::METHOD_GET, '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
+
+        $this->assertSame(0, $result->count());
+    }
+
+    public function testDisplayEditLinkWithAnimateurUser()
+    {
+        $this->authenticateAsAdherent($this->client, 'jacques.picard@en-marche.fr');
+        $crawler = $this->client->request(Request::METHOD_GET, '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
+
+        $this->assertSame(10, $result->count());
+    }
+
+    public function testDisplayEditLinkWithNormaleUser()
+    {
+        $this->authenticateAsAdherent($this->client, 'francis.brioul@yahoo.com');
+        $crawler = $this->client->request(Request::METHOD_GET, '/comites/en-marche-paris-8');
+
+        $this->assertResponseStatusCode(Response::HTTP_OK, $this->client->getResponse());
+
+        $result = $crawler->filter('.feed-rc-dropdown');
+
+        $this->assertSame(5, $result->count());
+    }
+
+    public function testEditMessage()
+    {
+        $committee = $this->manager->getRepository(Committee::class)->findOneBy(['slug' => 'en-marche-paris-8']);
+        $messages = $this->manager->getRepository(CommitteeFeedItem::class)->findMostRecentFeedEvent($committee->getUuid());
+        $this->authenticateAsAdherent($this->client, 'jacques.picard@en-marche.fr');
+
+        $crawler = $this->client->request(Request::METHOD_GET, '/comites/en-marche-paris-8/timeline/'.$messages->getId().'/edit');
+        $this->isSuccessful($this->client->getResponse());
+        $form = $crawler->selectButton('committee_feed_message_send')->form();
+        $this->assertSame($messages->getContent(), $form->get('committee_feed_message[content]')->getValue());
+        $form->setValues(['committee_feed_message[content]' => $messages->getContent().' test']);
+        $this->client->submit($form);
```

Comment:
Adding some blank line between request and assertations would be great for readability.